minetest/minetest
# Load media from subfolders

- This PR implements a requested feature.
- The dirs are added recursively. A media file is not loaded if a media file with the same name is in the parent directory if that is still a valid dir. Everything else in the priority should be undefined. (Apart from files in `textures/server` and `subgame/textures` being loaded first, in this order and being prioritized in this order.)
- How does the PR work?
- Closed #7818.

## To do

This PR is a Ready for Review.

## How to test

- Start a game in devtest and see how `dirt_with_grass` still looks normal.
- (Alternatively use https://github.com/DS-Minetest/smtt.)
- Rename subfolders (eg. add a `.` in the name) and play around and see if everything works as expected.
